process/security_baseline.md

```diff
@@ -0,0 +1,253 @@
+The initiative was one of the outcomes of the October 2023 Linux Foundation Member Summit. Making open source software more secure is one of the top priorities across the Linux Foundation. OpenSSF is leading the charge. 
+
+In the United States, open source software is used across all critical infrastructure sectors defined by CISA (Cybersecurity and Infrastructure Security Agency), for example, health care, defense, financial services, utilities, telecommunications, etc. Open source security directly impacts national security, economics and social stability. Enhancing open source security is imperative. NIST has published a [Secure Software Development Framework](https://csrc.nist.gov/Projects/ssdf) (SSDF) as a result of Executive Order (EO) 14028 on "[Improving the Nation's Cybersecurity](https://www.federalregister.gov/documents/2021/05/17/2021-10460/improving-the-nations-cybersecurity)".
+
+In the European Union, [Cyber Resiliency Act](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CONSIL:ST_17000_2023_INIT) (CRA) has declared a new role - Open Source Steward(Article 3(18a). The legislation is to ensure that consumers of products with digital features are protected, and manufacturers of these products are held accountable for product security. Open source security is the foundation for manufacturers to be compliant with the legislation.     
+
+## Success Criteria
+The success of the security baseline SHOULD be quantified and qualified in a few dimension:
+ * **Security Outcomes Adoption and Advancement**
+   * **Objective**: The security baseline promotes improved open source security outcomes.
+   * **Approach**: Utilize technical outcomes from the OSS community to implement, and verify the security baseline.
+   * **Collaborations**: Partner with other foundations to enhance OSS security through education, specifications, frameworks, and tooling.
+   * **Automation and Tooling**: Identify opportunities for automation and tooling to ease the burden on maintainers and contributors, facilitating secure by design and secure by default principles.
+ * **Increased Adoption Rate**
+   * **Objective**: Ensure the security baseline is widely adopted.
+   * **Metrics**:
+     * Three OpenSSF software-based pilot projects adopt the baseline by 9/15/2024.
+     * All OpenSSF software-based projects adopt the baseline by the end of 2024.
+     * At least two other LF foundations adopt the baseline by the end of 2024.
+* **Reduction in Security Findings**
+  * **Objective**: Measure the effectiveness of the baseline in improving a project’s security posture.
+  * **Metrics**:
+    * Use Scorecard statistics to track the reduction of Critical, High, and Medium risk findings through the project lifecycle. Benchmark is established at the adoption time. 
+    * Monitor the number of exceptions at both project and foundation levels, expecting a decline as projects mature and become more secure.
+* **Improved OSS Maintainer and Consumer Experience**
```

Comment:
Similar to my comment above, can we come up with a metric or two that would allow us to measure that maintainer/consumer experience has improved? I know this is incredibly hard, but finding some key measurable results may help in addressing future gaps etc.

Reply:
On the other hand, if we don't intend to measure this, I can see this point fitting under operating principles as well.

Reply:
@marcelamelara can we solicit feedback from maintainers during adoption about experience improvement measure? will a survey by the end of the adoption piloting be helpful?